assets/css/style-mobile.css

```diff
@@ -444,6 +446,44 @@ h4 {
   margin: 20px 0;
 }
 
+.web-nav {
+  display: none;
+  background: rgba(96, 112, 255, 0.88);
+  height: 100%;
+  width: 100%;
+  top: 0;
+  margin-left: -15px;
+  position: fixed;
+}
```

Comment:
- [ ] Nice work implementing the mobile menu :clap: However the space at the top of the options is too much compared to the Figma design. Kindly make sure you stick to the Figma design as much as possible.
### Your design :point_down: 
![Screenshot from 2023-06-21 21-51-07](https://github.com/kambereBr/Portfolio/assets/91142432/1a3afd82-711c-4382-8533-8acdfe12be00)
### Figma design :point_down: 
![Screenshot from 2023-05-15 17-26-08](https://github.com/kambereBr/Portfolio/assets/91142432/9508c5e4-e29d-4df4-9025-e8a6c7ce919a)

Reply:
Thanks @LionRouge1